docker-hub/configure-sign-in.md

```diff
@@ -0,0 +1,88 @@
+---
+description: Configure registry.json to require sign in
+keywords: authentication, registry.json, configure,
+title: Configure registry.json to require sign in
+---
+
+The `registry.json` file is a configuration file that allows administrators to
+specify the Docker organization  the user must belong to, and thereby ensure
+that the organization's settings are applied to the user's session. Docker
+Desktop installation requires admin access. In large enterprises where admin
+access is restricted, administrators can create a `registry.json` file and
+deploy it to the users’ machines using a device management software as part of
+the Docker Desktop installation process.
+
+When you deploy a `registry.json` file to a user’s machine, it prompts the user to sign into Docker Desktop. If a user doesn’t sign in, or tries to sign in using a different organization, other than the organization listed in the `registry.json` file, they will be denied access to Docker Desktop.
```

Comment:
> When you deploy a `registry.json` file to a user’s machine, it prompts the user to sign into Docker Desktop.

Well, it may be more `After you deploy a `registry.json` file to a user’s machine, it will prompt the user to sign into Docker Desktop.` or something similar?

Reply:
Aah, yes. That makes sense.

Reply:
Updated